partial-update-tests/existing/src/main/java/fixtures/bodystring/StringOperationClient.java

```diff
@@ -0,0 +1,298 @@
+        return this.serviceClient.putWhitespaceWithResponse(requestOptions);
+    }
+
+    /**
+     * Get String value when no string value is sent in response payload.
+     *
+     * <p><strong>Response Body Schema</strong>
+     *
+     * <pre>{@code
+     * String
+     * }</pre>
+     *
+     * @param requestOptions The options to configure the HTTP request before HTTP client sends it.
+     * @throws HttpResponseException thrown if the request is rejected by server.
+     * @return string value when no string value is sent in response payload.
+     */
+    @Generated
+    @ServiceMethod(returns = ReturnType.SINGLE)
+    public Response<BinaryData> getNotProvidedWithResponse(RequestOptions requestOptions) {
+        return this.serviceClient.getNotProvidedWithResponse(requestOptions);
+    }
+
+    /**
+     * Get value that is base64 encoded.
+     *
+     * <p><strong>Response Body Schema</strong>
+     *
+     * <pre>{@code
+     * byte[]
+     * }</pre>
+     *
+     * @param requestOptions The options to configure the HTTP request before HTTP client sends it.
+     * @throws HttpResponseException thrown if the request is rejected by server.
+     * @return value that is base64 encoded.
+     */
+    @Generated
+    @ServiceMethod(returns = ReturnType.SINGLE)
+    public Response<byte[]> getBase64EncodedWithResponse(RequestOptions requestOptions) {
+        return this.serviceClient.getBase64EncodedWithResponse(requestOptions);
+    }
+
+    /**
+     * Get value that is base64url encoded.
+     *
+     * <p><strong>Response Body Schema</strong>
+     *
+     * <pre>{@code
+     * Base64Url
+     * }</pre>
+     *
+     * @param requestOptions The options to configure the HTTP request before HTTP client sends it.
+     * @throws HttpResponseException thrown if the request is rejected by server.
+     * @return value that is base64url encoded.
+     */
+    @Generated
+    @ServiceMethod(returns = ReturnType.SINGLE)
+    public Response<BinaryData> getBase64UrlEncodedWithResponse(RequestOptions requestOptions) {
+        return this.serviceClient.getBase64UrlEncodedWithResponse(requestOptions);
+    }
+
+    /**
+     * Put value that is base64url encoded.
+     *
+     * <p><strong>Request Body Schema</strong>
+     *
+     * <pre>{@code
+     * Base64Url
+     * }</pre>
+     *
+     * @param stringBody string body.
+     * @param requestOptions The options to configure the HTTP request before HTTP client sends it.
+     * @throws HttpResponseException thrown if the request is rejected by server.
+     * @return the response.
+     */
+    @Generated
+    @ServiceMethod(returns = ReturnType.SINGLE)
+    public Response<Void> putBase64UrlEncodedWithResponse(BinaryData stringBody, RequestOptions requestOptions) {
+        return this.serviceClient.putBase64UrlEncodedWithResponse(stringBody, requestOptions);
+    }
+
+    public String manuallyAddedMethod() {
```

Comment:
Can you also add an annotation here and also include JavaDocs to ensure, it doesn't get overwritten?

Reply:
updated